spec/item_repository_spec.rb

```diff
@@ -118,17 +111,17 @@
 
   describe '#find_all_by_price_in_range(range)' do
     it 'returns empty array if no items in price range' do
-      details = MockData.items_as_hash(price_of: 10)
-      mock_data = MockData.items_as_mocks(details) { self }
+      details = MockData.items_as_hashes(unit_price: 10)
+      mock_data = MockData.items_as_mocks(self, details)
```

Comment:
Couple more here!

Reply:
Nvm -- I see this is needed.

Reply:
This one is actually needed to configure the prices for the data. Otherwise it's not needed to generate default hashes and pass them in

Reply:
Yes! I see that now, thanks!